manifests/p/Pinta/Pinta/3.0.0/Pinta.Pinta.locale.en-US.yaml

```
@@ -0,0 +1,134 @@
+  - Fixed an issue where the Pan tool's cursor could show up as a missing icon (#2013047)
+  - Fixed errors when saving a file that was opened with a missing or incorrect extension (#2013050)
+  - Fixed a bug where certain layer opacity settings could be incorrectly rounded (#2020596)
+  - Fixed bugs in the shape tools and the Lasso Select tool which prevented the last row and column of the image from being used (#467)
+  - Fixed issues where the Curves dialog could not easily edit existing control points (#1973602)
+  - Fixed a bug where dragging a control point in the Curves dialog could unexpectedly erase other control points (#1973602)
+  - Improved error handling when loading incompatible add-ins (#2047274)
+  - The Clone Stamp tool no longer resets the destination offset after each stroke (#2031257)
+  - Fixed potential errors when pasting in the text tool if the clipboard didn't contain text (#2047495)
+  - The text tool now supports pre-editing to display the intermediate characters entered by an input method (#2047445)
+  - Fixed layout issues in the effect dialogs (#2049937)
+  - Fixed a bug where the Flip Horizontal / Vertical items in the Image menu incorrectly activated the Layer menu's flip actions (#2051430)
+  - Fixed a bug where the uninstall Makefile target did not remove icons (#792)
+  - Fixed a potential crash on some platforms when entering characters in the text tool using an input method (#722)
+  - The angle picker widget now supports fractional angles (#807)
+  - Fixed issues with restoring saved settings in the Eraser tool (#839)
+  - Fixed dragging issues in the Curves adjustment dialog with modifiers such as Num Lock active (#871)
+  - Fixed a bug where the file picker dialogue would open the wrong directory after a failed save (#914)
+  - Fixed unexpected drawing behavior when using semi-transparent colors with the Paint Brush tool (#941)
+  - The Text tool now supports configuring whether antialiasing is enabled (#935)
+  - Fixed various artifacts in the shape tools, particularly with larger brush widths (#733, #955)
+  - Fixed an issue where the text tool did not immediately redraw after changes to the font or color of unfinalized text (#952, #975)
+  - Fixed an issue where the text tool could unexpectedly redraw existing text with the latest palette color, or fail to finalize uncommitted text (#1097, #1176)
+  - Fixed a bug where cut / paste operations did not behave as expected with complex selections (#951, #978)
+  - Fixed transparency behavior for several effects (#1184, #1229)
+  - Fixed issues with the Soften Portrait effect where the Softness and Lighting parameters had no effect (#1217)
+
+  ### Checksums
+
+  - Fingerprint: 5BDA 30D0 C2E9 85E5 9808  9E44 4CF8 0E80 E77C 5C35
+  - GPG Signature: gpg --verify pinta-3.0.0.tar.gz.asc
+  - Checksum: pinta-3.0.0.tar.gz.sha512
+ReleaseNotesUrl: https://github.com/PintaProject/PEinta/releases/tag/3.0
```

```suggestion
ReleaseNotesUrl: https://github.com/PintaProject/Pinta/releases/tag/3.0
```

Can't sure why there's an additional `E` ...